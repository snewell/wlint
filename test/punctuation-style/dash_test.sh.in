It’s good to write clearly--unless it’s for effect.

Spacing is important — no matter what the masses think.
