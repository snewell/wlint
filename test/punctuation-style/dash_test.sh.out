<stdin>-1:26 emdash.replace-double-hyphen
<stdin>-3:20 emdash.preceeding-space
<stdin>-3:21 emdash.following-space
